From: =?utf-8?q?Christian_G=C3=B6ttsche?= <cgzones@googlemail.com>
Date: Wed, 15 Jan 2020 15:25:54 +0100
Subject: drop unused examples directory

---
 CMakeLists.txt | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 067ab26..9902ded 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -628,7 +628,6 @@ CONFIGURE_FILE(config.h.in src/config.h)
 # Configs
 INSTALL(CODE "FILE(MAKE_DIRECTORY \$ENV{DESTDIR}${CONFDIR})")
 INSTALL(CODE "FILE(MAKE_DIRECTORY \$ENV{DESTDIR}${SHAREDIR})")
-INSTALL(CODE "FILE(MAKE_DIRECTORY \$ENV{DESTDIR}${EXAMPLESDIR})")
 INSTALL(CODE "FILE(MAKE_DIRECTORY \$ENV{DESTDIR}${LUALIBDIR})")
 INSTALL(CODE "FILE(MAKE_DIRECTORY \$ENV{DESTDIR}${PLUGINSDIR})")
 INSTALL(CODE "FILE(MAKE_DIRECTORY \$ENV{DESTDIR}${RULESDIR})")
@@ -735,4 +734,4 @@ if(_PVS_STUDIO)
 			ANALYZE ${PROJECT_NAME} rspamd-server rspamadm rspamc
 			OUTPUT FORMAT errorfile
 			LOG target_${PROJECT_NAME}.err)
-endif()
\ No newline at end of file
+endif()
