From: =?utf-8?q?Christian_G=C3=B6ttsche?= <cgzones@googlemail.com>
Date: Fri, 17 Jan 2020 16:59:00 +0100
Subject: drop ipv6 test by binding to random port

Forwarded: https://github.com/rspamd/rspamd/issues/3221
---
 src/libutil/addr.c | 25 ++-----------------------
 1 file changed, 2 insertions(+), 23 deletions(-)

diff --git a/src/libutil/addr.c b/src/libutil/addr.c
index 03e7e99..c7571a3 100644
--- a/src/libutil/addr.c
+++ b/src/libutil/addr.c
@@ -146,33 +146,12 @@ static void
 rspamd_ip_check_ipv6 (void)
 {
 	if (ipv6_status == RSPAMD_IPV6_UNDEFINED) {
-		gint s, r;
-		struct sockaddr_in6 sin6;
-		const struct in6_addr ip6_local = IN6ADDR_LOOPBACK_INIT;
-
-		s = socket (AF_INET6, SOCK_STREAM, 0);
+		gint s = socket (AF_INET6, SOCK_STREAM, 0);
 		if (s == -1) {
 			ipv6_status = RSPAMD_IPV6_UNSUPPORTED;
 		}
 		else {
-			/*
-			 * Some systems allow ipv6 sockets creating but not binding,
-			 * so here we try to bind to some local address and check, whether it
-			 * is possible
-			 */
-			memset (&sin6, 0, sizeof (sin6));
-			sin6.sin6_family = AF_INET6;
-			sin6.sin6_port = rspamd_random_uint64_fast () % 40000 + 20000;
-			sin6.sin6_addr = ip6_local;
-
-			r = bind (s, (struct sockaddr *)&sin6, sizeof (sin6));
-			if (r == -1 && errno != EADDRINUSE) {
-				ipv6_status = RSPAMD_IPV6_UNSUPPORTED;
-			}
-			else {
-				ipv6_status = RSPAMD_IPV6_SUPPORTED;
-			}
-
+			ipv6_status = RSPAMD_IPV6_SUPPORTED;
 			close (s);
 		}
 	}
