From: =?utf-8?q?Christian_G=C3=B6ttsche?= <cgzones@googlemail.com>
Date: Sun, 19 Jan 2020 20:55:56 +0100
Subject: enable lto/ipo

---
 cmake/Toolset.cmake | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/cmake/Toolset.cmake b/cmake/Toolset.cmake
index b6c8635..67d1d8a 100644
--- a/cmake/Toolset.cmake
+++ b/cmake/Toolset.cmake
@@ -144,7 +144,7 @@ set (CMAKE_CXX_FLAGS_RELWITHDEBINFO    "${CMAKE_CXX_FLAGS_RELEASE} -O2")
 
 if (CMAKE_BUILD_TYPE_UC MATCHES "COVERAGE")
     set (CMAKE_EXE_LINKER_FLAGS "${CMAKE_EXE_LINKER_FLAGS} --coverage")
-elseif (CMAKE_BUILD_TYPE_UC MATCHES "RELEASE")
+elseif (CMAKE_BUILD_TYPE_UC MATCHES "RELEASE" OR CMAKE_BUILD_TYPE_UC MATCHES "NONE")
     if (${CMAKE_VERSION} VERSION_GREATER "3.9.0")
         cmake_policy (SET CMP0069 NEW)
         include (CheckIPOSupported)
@@ -159,4 +159,4 @@ elseif (CMAKE_BUILD_TYPE_UC MATCHES "RELEASE")
 endif ()
 
 message (STATUS "Final CFLAGS: ${CMAKE_C_FLAGS} ${CMAKE_C_FLAGS_${CMAKE_BUILD_TYPE_UC}}")
-message (STATUS "Final CXXFLAGS: ${CMAKE_CXX_FLAGS} ${CMAKE_C_FLAGS_${CMAKE_BUILD_TYPE_UC}}")
\ No newline at end of file
+message (STATUS "Final CXXFLAGS: ${CMAKE_CXX_FLAGS} ${CMAKE_C_FLAGS_${CMAKE_BUILD_TYPE_UC}}")
