Description: Fix typo missmatch
Author: Viktor Szépe <viktor@szepe.net>
Origin: upstream, https://github.com/vstakhov/rspamd/commit/3f49cd695eb40bd00622806065e61ebd59471a17
Bug: https://github.com/vstakhov/rspamd/pull/1919
Forwarded: not-needed
Reviewed-By: Sebastien Badia <sbadia@debian.org>
Last-Update: 2018-02-19

--- rspamd-1.6.6.orig/src/libutil/map.c
+++ rspamd-1.6.6/src/libutil/map.c
@@ -344,7 +344,7 @@ rspamd_map_cache_cb (gint fd, short what
 		/* We have another update, so this cache element is obviously expired */
 		/* Important: we do not set cache availability to zero here */
 		MAP_RELEASE (cache_cbd->shm, "rspamd_http_map_cached_cbdata");
-		msg_debug_map ("cached data is now expired (gen missmatch) for %s", map->name);
+		msg_debug_map ("cached data is now expired (gen mismatch) for %s", map->name);
 		event_del (&cache_cbd->timeout);
 		g_slice_free1 (sizeof (*cache_cbd), cache_cbd);
 	}
